programs/server/Server.cpp

```diff
@@ -269,6 +269,7 @@ namespace ServerSetting
     extern const ServerSettingsUInt64 max_pending_mutations_to_warn;
     extern const ServerSettingsUInt64 max_pending_mutations_execution_time_to_warn;
     extern const ServerSettingsUInt64 max_parts_cleaning_thread_pool_size;
+    extern const ServerSettingsUInt64 max_named_collection_num_to_warn;
```

Comment:
I suggest to use the plural similar to `parts`, `tables` and `mutations` that are already present and remove `num`:
```suggestion
    extern const ServerSettingsUInt64 max_named_collection_to_warn;
```

Reply:
But I used exactly the same naming convention as the rest. It's actually `max_table_num_to_warn`. It's in singular and with `num`